preview/app/http/PreviewNoCacheFilter.scala

```diff
@@ -0,0 +1,13 @@
+package http
+
+import org.apache.pekko.stream.Materializer
+import play.api.mvc.{Filter, RequestHeader, Result}
+
+import scala.concurrent.{ExecutionContext, Future}
+
+// OBVIOUSLY this is only for the preview server
+// NOT to be used elsewhere...
```

Comment:
Would `/** */`-style comments not offer better IDE support for where NOT to use this class?

Reply:
moved this from elsewhere, I can make that change if you think this is important?

Reply:
If only moved not sure I have an opinion 🤷